src/tests/fixtures/resolver_context/ContextValueTypedAsString.invalid.ts.expected

```diff
@@ -0,0 +1,18 @@
+-----------------
+INPUT
+----------------- 
+/** @gqlType */
+export class Query {
+  /** @gqlField */
+  greeting(args: never, ctx: string): string {
+    return ctx;
+  }
+}
+
+-----------------
+OUTPUT
+-----------------
+src/tests/fixtures/resolver_context/ContextValueTypedAsString.invalid.ts:4:30 - error: Expected context parameter's type to be a type reference Grats validates that your context parameter is type-safe by checking all context values reference the same type declaration.
+
+4   greeting(args: never, ctx: string): string {
```

Comment:
Is not supporting `string` intentional? Or is it because some technical limitation?

Reply:
Technical limitation because we are tracking the _definition_ of the type, and primitives and literals won't work.